cmd/instance/instance_vnc.go

```diff
@@ -75,27 +78,27 @@ Duration follows Go's duration format (e.g. "30m", "1h", "24h")`,
 
 		err = waitEndpointReady(vnc.URI)
 		if err != nil {
-			utility.Error("VNC Console URL is not reachable: %s", err)
+			utility.Error("Console URL is not reachable: %s", err)
 			os.Exit(1)
 		}
 
-		utility.Info("Opening VNC in your default browser...")
+		utility.Info("Opening the Console in your default browser...")
 		time.Sleep(3 * time.Second)
 
 		// Open VNC in the browser
 		err = browser.OpenInBrowser(vnc.URI)
 		if err != nil {
-			utility.Error("Failed to open VNC URL in the browser: %s", err)
+			utility.Error("Failed to open the Console Access URL in the browser: %s", err)
 		} else {
-			utility.Info("VNC session is now active. You can access your instance's graphical interface.")
+			utility.Info("The Console Access Session is now active. You can access your instance's graphical interface.")
 		}
 	},
 }
 
 // endpointReady checks if the given URL endpoint is ready by sending a GET request
 // and returning true if the HTTP status code is not 503 or 40x
 func endpointReady(url string) bool {
-	utility.Info("New attempt to reach the VNC Console URI...")
+	utility.Info("New attempt to reach the Console URI...")
```

Comment:
Change to "New attempt to reach the console URI"

Actually, do we use "URL" and "URI" interchangeably?

Reply:
switched to URL and removed "the", I hope is correct English

Reply:
As above, I would keep it, but it's not critical